src/Tickets/Admin/Upsell.php

```diff
@@ -47,5 +48,33 @@ public function maybe_show_capacity_arf() {
 			],
 		] );
 	}
+	
+	/**
+	 * Maybe show upsell for Manual Attendees.
+	 * 
+	 * @since TBD
+	 */
+	public function maybe_show_manual_attendees() {
+		// If they already have ET+ activated, then bail.
+		if ( class_exists( 'Tribe__Tickets_Plus__Main' ) ) {
+			return;
+		}
+
+		echo '<div class="welcome-panel-column welcome-panel-extra">';
+		tribe( Upsell_Notice\Main::class )->render( [
+			'classes' => [
+				'tec-admin__upsell-event_tickets-manual_attendees'
+			],
+			'text'    => esc_html__( 'Manually add attendees with' , 'event-tickets' ),
```

Comment:
Maybe this is more related to the component itself, but is there any way that we can make the link a placeholder of the string? Not sure if it'll play well with all languages if we go this route.

Reply:
Yeah. The component treats the link as a totally different element. Would it be better to maybe use `sprint()` in this case, using 'Manually add attendees with %s', with the string just being a blank string, but a note for translators that `%s` is the name of the Event Tickets Plus plugin?

Reply:
Thanks! Yes, I think it can be better for i18n purposes.